Nalgebra Migration (#1066)
amethyst/amethyst: book/src/math.md

# Math

> Amethyst uses [nalgebra][na] under the hood as its math library and it's

Reviewer: ```suggestion
Amethyst uses [nalgebra][na] under the hood as its math library and it is
```

Author: May I ask why? We're using informal language all throughout the book and I have more in this paragraph alone. Why are you picking up on this particular instance?

> exposed for us to use as `amethyst::core::nalgebra`. As the documentation for

Reviewer: ```suggestion
re-exported for use under the `amethyst::core::nalgebra` namespace. As the documentation for
```
